deploy/kubernetes/clusterrole.yaml

```diff
@@ -104,6 +104,18 @@ rules:
   - list
   - patch
   - watch
+- apiGroups: ["rbac.istio.io"]
```

Comment:
@jmazzitelli @gbaufake we have new resource to give permission to. Would you mind update that at istio's side?